src/main/scala/mesosphere/marathon/event/http/HttpEventStreamActor.scala

```diff
@@ -0,0 +1,60 @@
+package mesosphere.marathon.event.http
+
+import akka.actor.{ PoisonPill, ActorRef, Props, Actor }
+import akka.event.EventStream
+import mesosphere.marathon.event._
+import mesosphere.marathon.event.http.HttpEventStreamActor._
+import org.apache.log4j.Logger
+import play.api.libs.json.Json
+
+/**
+  * A HttpEventStreamHandle is a reference to the underlying client http stream.
+  */
+trait HttpEventStreamHandle {
+  def connectionOpen: Boolean
+  def remoteAddress: String
```

Comment:
We do not have the clientIp, but only the remoteAddress. But good point, I will also replace clientIp->remoteAddress